tests/queries/0_stateless/03405_ssd_cache_layout.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# Tags: no-replicated-database
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
```

Comment:
Let's also add a comment on top which states that this is a test for GitHub issue #78314

Reply:
added in 1d06a0c061cc1822e6e1a57db4e36a3f938428ad